src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -782,6 +782,7 @@ public static partial class JsonMetadataServices
         public static System.Text.Json.Serialization.JsonConverter<sbyte> SByteConverter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<float> SingleConverter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<string> StringConverter { get { throw null; } }
+        public static System.Text.Json.Serialization.JsonConverter<System.TimeSpan> TimeSpanConverter { get { throw null; } }
```

Comment:
New APIs typically need to go through API review, but this seems pretty non-controversial. cc @layomia

Reply:
Yeah this is fine. Will include it when I take the diff of the source-gen APIs back to API review.